fe/fe-core/src/main/java/com/starrocks/sql/optimizer/Memo.java

```diff
@@ -206,10 +206,7 @@ private void mergeGroup(Group srcGroup, Group dstGroup) {
         // When some rule merge two groups to one group, or
         // the GroupExpressions of one group are all removed.
         // The group is empty, We should remove it.
-        List<Group> groups = getAllEmptyGroups();
-        for (Group group : groups) {
-            removeOneGroup(group);
-        }
+        removeAllEmptyGroup();
```

Comment:
Why change this?

Reply:
In the original way, a new empty group may appear, resulting in an error in the subsequent binder